### ISSUE TITLE
Single person should revel the estimation 

### ISSUE THREAD

**Issue by Sandeepjhanwar** [cschleiden/azure-boards-estimate#94]
Hello Team, 

Currently any team member can revel the estimation, it should be only to one person.

As now any one can revel complete estimation process need to start from first

Regards,

**Comment by wislerd** [comment]
Agree, only allowing a designated person such as session creator to reveal, especially when using the offline query would be helpful.
